scripts/validateContractDeployment.ts

```diff
@@ -148,11 +114,16 @@ function encodeRole(roleKey: string): string {
   return ethers.utils.keccak256(encoded);
 }
 
-async function checkRole(contractName: string, contractAddress: string, signalledRole: string): Promise<boolean> {
-  const rolesConfig = await roles(hre);
-  for (const [role, addresses] of Object.entries(rolesConfig)) {
+async function checkRole(
+  contractName: string,
+  contractAddress: string,
+  signalledRole: string,
+  requiredRolesForContracts: Record<string, string[]>
+): Promise<boolean> {
+  const rolesConfig = await hre.gmx.getRoles();
```

Comment:
@Call-Sol i probably missed it during PR review, there is no need to explicitly import config, you can use helpers like hre.gmx.getRoles or hre.gmx.getTokens instead